static/app/views/settings/project/projectKeys/list/index.spec.jsx

```diff
@@ -63,6 +63,156 @@ describe('ProjectKeys', function () {
     expect(expandButton).not.toBeInTheDocument();
   });
 
+  it('renders for default project', function () {
+    render(
+      <ProjectKeys
+        routes={[]}
+        organization={org}
+        params={{projectId: project.slug}}
+        project={TestStubs.Project({platform: 'other'})}
+      />
+    );
+
+    const allDsn = screen.queryAllByRole('textbox', {name: 'DSN URL'});
+    expect(allDsn.length).toBe(1);
+
+    const expandButton = screen.queryByRole('button', {name: 'Expand'});
+    const dsn = screen.queryByRole('textbox', {name: 'DSN URL'});
+    const minidumpEndpoint = screen.queryByRole('textbox', {
+      name: 'Minidump Endpoint URL',
+    });
+    const unrealEndpoint = screen.queryByRole('textbox', {
+      name: 'Unreal Engine 4 Endpoint URL',
+    });
+    const securityHeaderEndpoint = screen.queryByRole('textbox', {
+      name: 'Security Header Endpoint URL',
+    });
+
+    expect(expandButton).toBeInTheDocument();
+    expect(dsn).toHaveValue(projectKeys[0].dsn.public);
+    expect(minidumpEndpoint).toHaveValue(projectKeys[0].dsn.minidump);
+    // this is empty in the default ProjectKey
+    expect(unrealEndpoint).toHaveValue('');
+    expect(securityHeaderEndpoint).toHaveValue(projectKeys[0].dsn.security);
+  });
+
+  it('renders for javascript project', function () {
+    render(
+      <ProjectKeys
+        routes={[]}
+        organization={org}
+        params={{projectId: project.slug}}
+        project={TestStubs.Project({platform: 'javascript'})}
+      />
+    );
+
+    const expandButton = screen.queryByRole('button', {name: 'Expand'});
+    const dsn = screen.queryByRole('textbox', {name: 'DSN URL'});
```

Comment:
since we are expecting this to be in the page, we should use `getByRole` here 😉

```suggestion
    const dsn = screen.getByRole('textbox', {name: 'DSN URL'});
```